mira/dkg/resources/eiffelnames.py

```diff
@@ -0,0 +1,178 @@
+"""Get terms from eiffel names."""
```

Comment:
is this file needed?

Reply:
This file I created to create terms for eiffel